### DISCUSSION TITLE
Solved: 500 error, connect ECONNREFUSED 192.168.64.7:3001

### DISCUSSION THREAD

**Discussion by Majika007** [immich-app/immich#2260] [Q&A]
Hi. I have OpenMediaVault 6.3.7-5 (Shaitan) server (Debian). This is what I did: [https://immich.app/docs/install/portainer](https://immich.app/docs/install/portainer)
My "docker-compose.yml" from Github; I change only ".env => stack.env"
My ".env" from Github; I change only "UPLOAD_LOCATION=/srv/dev-disk-by-uuid-7e3eafd6-12d0-46fc-8b9b-5bac3721694a/Kepek"

**machine-learning log:**
[2023-04-16 12:33:49 +0000] [1] [INFO] Starting gunicorn 20.1.0
[2023-04-16 12:33:49 +0000] [1] [INFO] Listening at: http://0.0.0.0:3003 (1)
[2023-04-16 12:33:49 +0000] [1] [INFO] Using worker: gthread
[2023-04-16 12:33:50 +0000] [9] [INFO] Booting worker with pid: 9
[2023-04-16 12:33:50 +0000] [10] [INFO] Booting worker with pid: 10

**microservices log:**
[Nest] 1  - 04/16/2023, 12:32:37 PM     LOG [NestFactory] Starting Nest application...
[Nest] 1  - 04/16/2023, 12:32:39 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +1567ms
[Nest] 1  - 04/16/2023, 12:32:39 PM     LOG [InstanceLoader] BullModule dependencies initialized +1ms
[Nest] 1  - 04/16/2023, 12:32:39 PM     LOG [InstanceLoader] ConfigHostModule dependencies initialized +3ms
[Nest] 1  - 04/16/2023, 12:32:39 PM     LOG [InstanceLoader] DiscoveryModule dependencies initialized +2ms
[Nest] 1  - 04/16/2023, 12:32:39 PM     LOG [InstanceLoader] ConfigModule dependencies initialized +38ms
[Nest] 1  - 04/16/2023, 12:32:39 PM     LOG [InstanceLoader] BullModule dependencies initialized +2ms
[Nest] 1  - 04/16/2023, 12:32:39 PM     LOG [InstanceLoader] BullModule dependencies initialized +1ms
[Nest] 1  - 04/16/2023, 12:32:42 PM     LOG [InstanceLoader] TypeOrmCoreModule dependencies initialized +3049ms
[Nest] 1  - 04/16/2023, 12:32:42 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +3ms
[Nest] 1  - 04/16/2023, 12:32:42 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +2ms
[Nest] 1  - 04/16/2023, 12:32:42 PM    WARN [MetadataExtractionProcessor] Reverse geocoding is enabled
[Nest] 1  - 04/16/2023, 12:32:42 PM     LOG [MetadataExtractionProcessor] Initializing Reverse Geocoding
[Nest] 1  - 04/16/2023, 12:32:42 PM     LOG [InstanceLoader] InfraModule dependencies initialized +41ms
[Nest] 1  - 04/16/2023, 12:32:42 PM     LOG [InstanceLoader] DomainModule dependencies initialized +2ms
[Nest] 1  - 04/16/2023, 12:32:42 PM     LOG [InstanceLoader] MicroservicesModule dependencies initialized +2ms
[Nest] 1  - 04/16/2023, 12:32:42 PM     LOG [NestApplication] Nest application successfully started +84ms
[Nest] 1  - 04/16/2023, 12:32:42 PM     LOG [ImmichMicroservice] Running Immich Microservices in PRODUCTION environment - version 1.53.0 - Listening on port: 3002
[Nest] 1  - 04/16/2023, 12:33:16 PM     LOG [MetadataExtractionProcessor] Reverse Geocoding Initialized

**postres log:**
PostgreSQL Database directory appears to contain a database; Skipping initialization
2023-04-16 12:31:33.144 UTC [1] LOG:  starting PostgreSQL 14.7 (Debian 14.7-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
2023-04-16 12:31:33.181 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2023-04-16 12:31:33.181 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2023-04-16 12:31:33.555 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2023-04-16 12:31:34.246 UTC [27] LOG:  database system was shut down at 2023-04-16 12:25:44 UTC
2023-04-16 12:31:34.567 UTC [1] LOG:  database system is ready to accept connections

**redis log:**
1:C 16 Apr 2023 12:31:23.460 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
1:C 16 Apr 2023 12:31:23.460 # Redis version=6.2.11, bits=64, commit=00000000, modified=0, pid=1, just started
1:C 16 Apr 2023 12:31:23.460 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
1:M 16 Apr 2023 12:31:23.463 * monotonic clock: POSIX clock_gettime
1:M 16 Apr 2023 12:31:23.466 * Running mode=standalone, port=6379.
1:M 16 Apr 2023 12:31:23.467 # Server initialized
1:M 16 Apr 2023 12:31:23.467 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. Being disabled, it can can also cause failures without low memory condition, see https://github.com/jemalloc/jemalloc/issues/1328. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
1:M 16 Apr 2023 12:31:23.469 * Ready to accept connections
1:M 16 Apr 2023 12:41:46.166 * 100 changes in 300 seconds. Saving...
1:M 16 Apr 2023 12:41:46.169 * Background saving started by pid 20
20:C 16 Apr 2023 12:41:46.243 * DB saved on disk
20:C 16 Apr 2023 12:41:46.244 * RDB: 0 MB of memory used by copy-on-write
1:M 16 Apr 2023 12:41:46.269 * Background saving terminated with success

**server log:**
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/oauth/config, POST} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/oauth/callback, POST} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/oauth/link, POST} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/oauth/unlink, POST} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RoutesResolver] SearchController {/search}: +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/search, GET} route +3ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/search/config, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/search/explore, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RoutesResolver] ServerInfoController {/server-info}: +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/server-info, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/server-info/ping, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/server-info/version, GET} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/server-info/stats, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RoutesResolver] ShareController {/share}: +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/share, GET} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/share/me, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/share/:id, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/share/:id, DELETE} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/share/:id, PATCH} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RoutesResolver] SystemConfigController {/system-config}: +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/system-config, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/system-config/defaults, GET} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/system-config, PUT} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/system-config/storage-template-options, GET} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RoutesResolver] UserController {/user}: +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/user, GET} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/user/info/:userId, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/user/me, GET} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/user, POST} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/user/count, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/user/:userId, DELETE} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/user/:userId/restore, POST} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/user, PUT} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/user/profile-image, POST} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/user/profile-image/:userId, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RoutesResolver] AssetController {/asset}: +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/upload, POST} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/download/:assetId, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/download-files, POST} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/download-library, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/file/:assetId, GET} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/thumbnail/:assetId, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/curated-objects, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/curated-locations, GET} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/search-terms, GET} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/search, POST} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/count-by-time-bucket, POST} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/count-by-user-id, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset, GET} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/time-bucket, POST} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/:deviceId, GET} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/assetById/:assetId, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/:assetId, PUT} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset, DELETE} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/check, POST} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/exist, POST} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/shared-link, POST} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/shared-link/add, PATCH} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/asset/shared-link/remove, PATCH} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RoutesResolver] TagController {/tag}: +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/tag, POST} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/tag, GET} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/tag/:id, GET} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/tag/:id, PATCH} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/tag/:id, DELETE} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RoutesResolver] AlbumController {/album}: +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/album/count-by-user-id, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/album, POST} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/album/:albumId/users, PUT} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/album/:albumId/assets, PUT} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/album/:albumId, GET} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/album/:albumId/assets, DELETE} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/album/:albumId, DELETE} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/album/:albumId/user/:userId, DELETE} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/album/:albumId, PATCH} route +1ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/album/:albumId/download, GET} route +2ms
[Nest] 1  - 04/16/2023, 12:47:07 PM     LOG [RouterExplorer] Mapped {/album/create-shared-link, POST} route +1ms
[Nest] 1  - 04/16/2023, 12:47:08 PM     LOG [SearchService] Running bootstrap
Request #1681649228041: Request to Node 0 failed due to "ENOTFOUND getaddrinfo ENOTFOUND typesense"
Request #1681649228041: Sleeping for 4s and then retrying request...
Request #1681649228041: Request to Node 0 failed due to "ECONNREFUSED connect ECONNREFUSED 192.168.64.9:8108"
Request #1681649228041: Sleeping for 4s and then retrying request...

**typesense log:** 
I20230416 12:42:27.488543     1 typesense_server_utils.cpp:357] Starting Typesense 0.24.0
I20230416 12:42:27.488715     1 typesense_server_utils.cpp:360] Typesense is using jemalloc.
I20230416 12:42:27.490388     1 typesense_server_utils.cpp:409] Thread pool size: 16
I20230416 12:42:27.505677     1 store.h:63] Initializing DB by opening state dir: /data/db
I20230416 12:42:27.692771     1 store.h:63] Initializing DB by opening state dir: /data/meta
I20230416 12:42:28.050406     1 ratelimit_manager.cpp:475] Loaded 0 rate limit rules.
I20230416 12:42:28.050470     1 ratelimit_manager.cpp:476] Loaded 0 rate limit bans.
I20230416 12:42:28.050932     1 typesense_server_utils.cpp:504] Starting API service...
I20230416 12:42:28.051766   118 typesense_server_utils.cpp:255] Since no --nodes argument is provided, starting a single node Typesense cluster.
I20230416 12:42:28.052443     1 http_server.cpp:178] Typesense has started listening on port 8108
I20230416 12:42:28.052978   119 batched_indexer.cpp:124] Starting batch indexer with 16 threads.
I20230416 12:42:28.061776   119 batched_indexer.cpp:129] BatchedIndexer skip_index: -9999
I20230416 12:42:28.081416   118 server.cpp:1045] Server[braft::RaftStatImpl+braft::FileServiceImpl+braft::RaftServiceImpl+braft::CliServiceImpl] is serving on port=8107.
I20230416 12:42:28.081504   118 server.cpp:1048] Check out http://0fa8554c5fd4:8107 in web browser.
I20230416 12:42:28.082929   118 raft_server.cpp:65] Nodes configuration: 192.168.64.9:8107:8108
I20230416 12:42:28.083380   118 raft_server.cpp:106] Snapshot does not exist. We will remove db dir and init db fresh.
I20230416 12:42:28.086050   118 store.h:298] rm /data/db success
I20230416 12:42:28.086650   118 store.h:63] Initializing DB by opening state dir: /data/db
I20230416 12:42:28.353155   118 store.h:322] DB open success!
I20230416 12:42:28.353191   118 raft_server.cpp:490] Loading collections from disk...
I20230416 12:42:28.353225   118 collection_manager.cpp:159] CollectionManager::load()
I20230416 12:42:28.353541   118 auth_manager.cpp:34] Indexing 0 API key(s) found on disk.
I20230416 12:42:28.353565   118 collection_manager.cpp:179] Loading upto 8 collections in parallel, 1000 documents at a time.
I20230416 12:42:28.353580   118 collection_manager.cpp:188] Found 0 collection(s) on disk.
I20230416 12:42:28.354291   118 collection_manager.cpp:278] Loaded 0 collection(s).
I20230416 12:42:28.355024   118 collection_manager.cpp:282] Initializing batched indexer from snapshot state...
I20230416 12:42:28.355065   118 raft_server.cpp:497] Finished loading collections from disk.
I20230416 12:42:28.356150   118 log.cpp:677] Use murmurhash32 as the checksum type of appending entries
I20230416 12:42:28.356328   118 log.cpp:1158] log load_meta /data/state/log/log_meta first_log_index: 1 time: 139
I20230416 12:42:28.356410   118 log.cpp:1098] load open segment, path: /data/state/log first_index: 1
I20230416 12:42:28.369455   118 raft_meta.cpp:521] Loaded single stable meta, path /data/state/meta term 8 votedfor 0.0.0.0:0:0 time: 12896
I20230416 12:42:28.369668   118 raft_meta.cpp:521] Loaded single stable meta, path /data/state/meta term 8 votedfor 0.0.0.0:0:0 time: 24
I20230416 12:42:28.369690   118 node.cpp:576] node default_group:192.168.64.9:8107:8108 init, term: 8 last_log_id: (index=1,term=3) conf: 192.168.16.9:8107:8108 old_conf: 
I20230416 12:42:28.369732   118 raft_server.cpp:129] Node last_index: 1
I20230416 12:42:28.369745   118 typesense_server_utils.cpp:306] Typesense peering service is running on 192.168.64.9:8107
I20230416 12:42:28.369755   118 typesense_server_utils.cpp:307] Snapshot interval configured as: 3600s
I20230416 12:42:28.369764   118 typesense_server_utils.cpp:308] Snapshot max byte count configured as: 4194304
W20230416 12:42:28.369776   118 controller.cpp:1454] SIGINT was installed with 1
I20230416 12:42:28.369799   118 raft_server.cpp:545] Term: 8, last_index index: 1, committed_index: 0, known_applied_index: 0, applying_index: 0, queued_writes: 0, pending_queue_size: 0, local_sequence: 0
W20230416 12:42:28.369825   118 raft_server.cpp:569] Single-node with no leader. Resetting peers.
W20230416 12:42:28.369835   118 node.cpp:894] node default_group:192.168.64.9:8107:8108 set_peer from 192.168.16.9:8107:8108 to 192.168.64.9:8107:8108
I20230416 12:42:28.394405   118 raft_meta.cpp:546] Saved single stable meta, path /data/state/meta term 9 votedfor 0.0.0.0:0:0 time: 24535
E20230416 12:42:28.820235     7 backward.hpp:4199] Stack trace (most recent call last) in thread 7:
E20230416 12:42:28.820286     7 backward.hpp:4199] #7    Object "", at 0xffffffffffffffff, in 
E20230416 12:42:28.820294     7 backward.hpp:4199] #6    Object "/usr/lib/x86_64-linux-gnu/libc-2.31.so", at 0x7f415c49f132, in __clone
E20230416 12:42:28.820300     7 backward.hpp:4199] #5    Object "/usr/lib/x86_64-linux-gnu/libpthread-2.31.so", at 0x7f415c6db608, in start_thread
E20230416 12:42:28.820310     7 backward.hpp:4199] #4    Object "/opt/typesense-server", at 0x84d328, in bvar::detail::SamplerCollector::sampling_thread(void*)
E20230416 12:42:28.820317     7 backward.hpp:4199] #3    Object "/opt/typesense-server", at 0x84b818, in bvar::detail::SamplerCollector::run()
E20230416 12:42:28.820322     7 backward.hpp:4199] #2  | Source "/usr/local/include/bvar/passive_status.h", line 62, in get_value
E20230416 12:42:28.820327     7 backward.hpp:4199]       Source "/usr/local/include/bvar/passive_status.h", line 140, in take_sample [0x7a4323]
E20230416 12:42:28.820335     7 backward.hpp:4199] #1    Source "/opt/braft-80d97b2475b3c0afca79c19b64d46bb665d704f4/src/braft/util.cpp", line 215, in counter_percentile [0x7e17df]
E20230416 12:42:28.820340     7 backward.hpp:4199] #0    Source "/usr/local/include/bvar/detail/percentile.h", line 294, in snprintf [0x7e51e5]
Illegal instruction (Illegal operand [0x7e51e5])
E20230416 12:42:28.935232     7 typesense_server.cpp:102] Typesense 0.24.0 is terminating abruptly.

web log:
Listening on 0.0.0.0:3000

Please, help me! :(

**Comment by bo0tzz** [comment]
```
Illegal instruction (Illegal operand [0x7e51e5])
E20230416 12:42:28.935232 7 typesense_server.cpp:102] Typesense 0.24.0 is terminating abruptly.
```

Your CPU doesn't support some instructions that are required by typesense. You can comment it out from the docker-compose.yml and set `TYPESENSE_ENABLED=false` to disable it.

**Reply by Majika007** [reply]
Thank You Very much!!!!  
(Note: the in ".env" file..)
